pithos/pithos.py

```diff
@@ -520,6 +520,7 @@ def user_pause(self, *ignore):
 
     def pause(self):
         self.playing = False
+        self.destroy_ui_loop()
```

Comment:
The loop dies 1 sec after it return False. Destroying the loop kills it right then. So I guess it really doesn't matter.